helloworld.cs

```diff
@@ -1 +1,7 @@
-Test
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        System.Console.WriteLine("Hello, World!");
```

Comment:
I didn't understand at first what this variable `System` meant. Could we discuss this in 15 minutes? I can come by your office after lunch.